index.bs

```diff
@@ -1039,23 +1039,25 @@ dictionary ReadableStreamDefaultReadResult {
 Instances of {{ReadableStreamDefaultReader}} are created with the internal slots described in the
 following table:
 
-<table>
+<table dfn-for="ReadableStreamDefaultReader">
  <thead>
   <tr>
    <th>Internal Slot
    <th>Description (<em>non-normative</em>)
  <tbody>
   <tr>
-   <td>\[[closedPromise]]
+   <td><dfn>\[[closedPromise]]</dfn>
    <td class="non-normative">A promise returned by the reader's
-   {{ReadableStreamDefaultReader/closed}} getter
+   {{ReadableStreamDefaultReader/closed}} getter.
+   This fulfills the [=ReadableStreamGenericReader/[[closedPromise]]=] contract.
```

Comment:
This is probably not a legal construct. 😛

Reply:
There's a few ways to do this. I think it'd be best to delete the [[closedPromise]] and [[stream]] rows from this table, and then add a sentence after the table saying something like "They additionally have the internal slots described in the section XXX".